modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -2485,13 +2544,19 @@ private void releaseCell(T cell) {
         }
     }
 
+
     /**
-     * This method is an experts-only method - if the requested index is not
-     * already an existing visible cell, it will create a cell for the
-     * given index and insert it into the sheet. From that point on it will be
-     * unmanaged, and is up to the caller of this method to manage it.
+     * Creates and returns a new cell for the given index.
+     * <p>
+     * This method is an experts-only method - if the requested index is not already an existing visible cell, it will
```

Comment:
"This method is an experts-only method" Not sure how fitting this remark is. If it's supposed to be a warning, write the warning itself (that is, why is it "expert-only?). If there are pitfalls, explain them (and maybe how to deal with them).